lib-android/build.gradle

```diff
@@ -51,19 +53,4 @@ task androidSourcesJar(type: Jar) {
 
 artifacts {
   archives androidSourcesJar
-}
-
-uploadArchives {
-  dependsOn 'testDebugUnitTest'
-  repositories {
-    mavenDeployer {
-      pom {
-        groupId = 'com.devinci.lib'
-        artifactId = 'android'
-        version = android.defaultConfig.versionName
-      }
-      repository(url: 'file://' + new File(System.getProperty('user.home'),
-          '.m2/repository').absolutePath)
-    }
-  }
-}
+}
```

Comment:
It should contain empty line at the end. Do you have "Ensure line feed at file end on Save" enabled in AS?